pubspec.lock

```diff
@@ -5,254 +5,298 @@ packages:
     dependency: transitive
     description:
       name: _fe_analyzer_shared
-      url: "https://pub.dartlang.org"
+      sha256: "4897882604d919befd350648c7f91926a9d5de99e67b455bf0917cc2362f4bb8"
```

Comment:
lock file shall not be updated unless we aim to upgrade packages as it can break the app due to patch upgrades in transitive dependencies(see https://github.com/CCExtractor/beacon/issues/203). Please remove unnecessary changes and only keep changes for the refresh indicator

Reply:
Right, so I should essentially revert any lock file changes right?
Also, the hashes were added automatically, can you suggest how I should sort out this issue

Reply:
regarding hashes, I think it was introduced later by dart here(https://github.com/dart-lang/pub/pull/3482)
Earlier dependencies were not locked to hashes but now they're that's why you see so many autogen hash.
I think that's something we can take in some other PR, where we do 2 things, full package upgrade and make new lock file.

For now, i would suggest just using the dependency of the refresh indicator(as that's the only new package you added), and leave the rest as they're in main branch. 

For flutter version, which flutter are you using? Flutter 3+ works fine with the packages.